src/mixins/filing-template-mixin.ts

```diff
@@ -1468,14 +1468,15 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
   }
 
   /**
-   * Fixes addresses by deleting the type in case it was null.
+   * Fixes addresses by deleting the type in case it is null. Also fixes null officer email.
    * @param orgPeople The array of orgs/people
-   * @returns the array of orgs/people after addresses been fixed
+   * @returns the array of orgs/people with fixed address types and email
    */
   fixNullAddressType (orgPeople: OrgPersonIF[]): OrgPersonIF[] {
     return orgPeople.map(p => {
       if (p.deliveryAddress?.addressType === null) delete p.deliveryAddress.addressType
       if (p.mailingAddress?.addressType === null) delete p.mailingAddress.addressType
+      if (p.officer?.email === null) delete p.officer.email
```

Comment:
[Party schema](https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/party.json) shows this:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/d484e7a3-3b65-4a3b-9838-9c2603037d49)

[JSON schema reference](https://json-schema.org/understanding-json-schema/reference/string#string) shows this:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/1bf1d2c6-a1ec-4454-bb74-0081e9f0c477)